slyd/slyd/splash/css_utils.py

```diff
@@ -1,12 +1,37 @@
 import re
 import urllib
+import six
+import htmlentitydefs
```

Comment:
Can you change this import to `import six.moves.html_entities as htmlentitydefs`